cvat/apps/dataset_manager/formats/README.md

```diff
@@ -536,14 +536,13 @@ archive.zip/
   - `score` (number) - the annotation `score` field
   - arbitrary attributes - will be stored in the `attributes` annotation section
 
-
-*Note*: there is also a [support for COCO keypoints over Datumaro](https://github.com/openvinotoolkit/cvat/issues/2910#issuecomment-726077582)
```

Comment:
OK. Probably it is a good idea to revert all files which you don't want to change.